src/main/java/core/basesyntax/shop/strategy/FruitShopStrategyImpl.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.shop.strategy;
+
+import core.basesyntax.shop.model.Fruit;
+import core.basesyntax.shop.service.FruitShopService;
+import core.basesyntax.shop.service.Operations;
+import java.lang.reflect.Method;
+
+public class FruitShopStrategyImpl implements FruitShopStrategy {
+    private static final String BALANCE = Operations.B.name().toLowerCase();
+    private static final String SUPPLY = Operations.S.name().toLowerCase();
+    private static final String RETURN_BACK = Operations.R.name().toLowerCase();
+    private static final String PURCHASE = Operations.P.name().toLowerCase();
```

Comment:
Seems that something mentors will tell you to change cause of we need to use map.
Why? Cause of in theory video we was told about awesome java Spring framework.

Reply:
it was just one of the possible sollutions